trieste/bayesian_optimizer.py

```diff
@@ -707,7 +718,6 @@ def optimize(
                             models,
                             query_points,
                             tagged_output,
-                            initial_model_fitting_timer,
```

Comment:
not needed anymore as logged separately now

Reply:
:+1: